src/ape/managers/project.py

```diff
@@ -1891,6 +1891,12 @@ def reconfigure(self, **overrides):
         self.account_manager.test_accounts.reset()
 
     def extract_manifest(self) -> PackageManifest:
+        # Ensure it compiled (at least attempt).
+        try:
+            self.load_contracts()
+        except CompilerError as err:
+            logger.debug(err)
```

Comment:
Should this be higher? Maybe warning

Reply:
perhaps! I am just worried about dependencies trying to extract manifests from that aren't meant to be compiled.

Reply:
i suppose they could just `.manifest` in that case.

Reply:
I changed it to warning but added a comment. let me know if that makes sense, along with my concerns.

Reply:
> perhaps! I am just worried about dependencies trying to extract manifests from that aren't meant to be compiled.

Wonder if it makes more sense as like `.enrich_manifest` or `.fully_compiled_manifest`, or something